README.md

```diff
@@ -1 +1,35 @@
+Portfolio
+
+The project's goal is to clone a design from a portfolio with html and css. Applying new tools such us Flexbox, Figma, and as well as using best practices for html and css. This project is the start of a micronaut future portfolio!
+
+![Alt text](/images/screenshot_1.png?raw=true)
+
+Built With
+- HTML, CSS, JS
+- Linters, Node.js, Git, Flexbox
+
+[Live Demo](https://mateo951.github.io/portfolio/)
+
+To get a local copy up and running follow these simple example steps.
+- On your terminal write the following command to clone the repository
+    - git clone git@github.com:mateo951/portfolio.git
```

Comment:
convert into inline command by adding `command` like that

Reply:
`git commit -m "Add the inline command that was requested from my reviewer to my README file"` :weight_lifting: